bin/main.rb

```diff
@@ -0,0 +1,110 @@
+
+require_relative '../lib/tic_tac_toe.rb'
+
+def msg(text)
+  puts text
+end
+
+def ask_input(message)
+  msg(message)
+  result= gets.chomp
+  result
+end
+
+def show_board(board)
+  puts " #{board[0] == 10 ? 'X' : board[0]} | #{board[1] == 10 ? 'X' : board[1]} | #{board[2] == 10 ? 'X' : board[2]}"
+  puts " #{board[3] == 10 ? 'X' : board[3]} | #{board[4] == 10 ? 'X' : board[4]} | #{board[5] == 10 ? 'X' : board[5]}"
+  puts " #{board[6] == 10 ? 'X' : board[6]} | #{board[7] == 10 ? 'X' : board[7]} | #{board[8] == 10 ? 'X' : board[8]}"
+end
```

Comment:
- Objects in the game should communicate explicitly through messages(methods) not implicitly through global methods. You can associate these methods with an existing class, or create a sophisticated one for them. :nerd_face: